backend/app/adapter/sqldb/table/progress.go

```diff
@@ -0,0 +1,16 @@
+package table
+
+// Progress represents database table columns for 'progress' table
+var Progress = struct {
+	TableName        string
+	ColumnIncidentId string
```

Comment:
```suggestion
	ColumnIncidentID string
```
You have to change it also here